harvard-university-of-worcester.csl

```diff
@@ -276,7 +276,7 @@
       </group>
     </layout>
   </citation>
-  <bibliography et-al-use-first="4" et-al-min="5" hanging-indent="false">
+  <bibliography et-al-use-first="20" et-al-min="19" hanging-indent="false">
```

Comment:
"et-al-use-first" should always be a lower number than "et-al-min". It's the number of authors to print when "et al." abbreviation kicks in, so by definition you're hiding some authors (the "et alia"/"and others" ones).